### ISSUE TITLE
Empy layer is giving an extra space with the icon

### ISSUE THREAD

**Issue by piensaenpixel** [CartoDB/cartodb#13486]
@ramiroaznar could we prioritize this one in the RT?

![image](https://user-images.githubusercontent.com/1214201/35684495-4e60641c-0767-11e8-9974-0203dc4549a2.png)

cc @jesusbotella

**Comment by jesusbotella** [comment]
This comes from the empty layers' DataLayerView rollback that was addressed here: https://github.com/CartoDB/cartodb/pull/13432.

I forgot to remove the code that shows the empty layer node in the template, so removing this code should suffice: https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb3/editor/layers/layer-views/data-layer.tpl#L70-L72

**Comment by piensaenpixel** [comment]
hey @jesusbotella do you think this too can be related to?

https://github.com/CartoDB/cartodb/issues/13478

**Comment by jesusbotella** [comment]
Yes, it is related to this one. The change of disabling the analysis tab was introduced in this commit: https://github.com/CartoDB/cartodb/pull/13377/commits/b0b9bfc3bcd238c538ec21b45079b212305b9aca

**Comment by ramiroaznar** [comment]
Can you take care @Jesus89?

**Comment by jesusbotella** [comment]
Yes, I think so!

**Comment by jesusbotella** [comment]
@piensaenpixel It is now deployed in `ded11` to test.
